WebJobs.Script.proj

```diff
@@ -168,6 +168,11 @@
     <xunit Assemblies="@(UnitTestAssemblies)" ExcludeTraits="$(ExcludedTestTraits)" />
   </Target>
 
+  <Target Name="MochaTest" AfterTargets="TestBuild">
+    <Exec Command="npm i mocha chai mocha-appveyor-reporter" />
```

Comment:
It'd be good to get the mocha tests to run serially with the xunit tests. That can be done as a follow up.